stubs/requests/requests/auth.pyi

```diff
@@ -1,6 +1,7 @@
 from typing import Any
 
 from . import cookies, models, utils
+from .compat import basestring as basestring
```

Comment:
Does this need to be re-exported from this module? It would be a bit weird to do `from requests.auth import basestring`, surely? Shouldn't end users be importing `basestring` from `.compat`?

Reply:
This was a stubtest related error locally which is why it was added and re-exported. I don't disagree with you but I don't want to avoid doing what the runtime actually has